src/Controls/src/Core/ListView.cs

```diff
@@ -458,6 +458,8 @@ protected override void UnhookContent(Cell content)
 			if (content == null || !_logicalChildren.Contains(content))
 				return;
 			var index = _logicalChildren.IndexOf(content);
+			if (index == -1)
+				return;
 			_logicalChildren.Remove(content);
```

Comment:
Use `RemoveAt` to remove another lookup?
```suggestion
			_logicalChildren.RemoveAt(index);
```